faseb-journal.csl

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize-with=", " names-delimiter=", " page-range-format="expanded" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>FASEB Journal</title>
+    <title-short>FASEB</title-short>
```

Comment:
the main title should clearly be The FASEB Journal (but with the "The"), I don't think there's any disagreement. @rmzelle was wondering whether we should have an additional field to store long titles of journals (e.g. containing association information or subtitles), but that's a discussion for another place.
Right now, if you'd add a summary field that contains the long title that's be cool. Also see Rintze's other requests.